particle-iot/particle-cli
# bugfix/ch52548/tinker-for-b5som

## Description

`particle flash <device> tinker` fails when targeting a `b5som`


## How to Test

1. plugin you device
2. run `particle flash <device> tinker`
3. verify the app is flashed OTA


## Completeness
